packages/terra-site/src/cerner-mock-theme.scss

```diff
@@ -571,4 +571,32 @@
   --terra-status-view-actions-button-margin-left: 0.357rem;
   --terra-status-view-actions-button-margin-right: 0.357rem;
   --terra-status-view-actions-button-margin-top: 0.714rem;
+
+  //terra-tag
+  --terra-tag-border-style: dotted;
+  --terra-tag-font-size: 20px;
```

Comment:
Can you change px to rem here and below to remain consistent?

Reply:
Fixed it.